docs/source/runtime.rst

```diff
@@ -292,24 +292,25 @@ For additional information, please contact the GEOPM team.
 Linux Power Management
 ^^^^^^^^^^^^^^^^^^^^^^
 
-It's crucial to note that other Linux mechanisms for power management can
-interfere with GEOPM, which must be disabled. It's recommended to disable the
-``intel_pstate`` kernel driver by modifying the kernel command line through
-grub2 or your system bootloader by adding:
+It's crucial to note that other Linux mechanisms for CPU power management can
+interfere with optimization objectives of GEOPM agents. When deploying an agent
+that controls CPU frequency or power limits, it's recommended that the generic
+scaling govermernor ``userspace`` is selected.
```

Comment:
I think it's important to still mention intel_pstate here.  It may interfere with your results depending on what you're trying to do.  We should maybe mention userspace alongside performance.

Reply:
performance should be mentioned for when userspace is not available.

consider moving HPC tuning tips to a separate page.